tests/bpftrace.cpp

```diff
@@ -222,34 +222,41 @@ TEST(bpftrace, add_probes_wildcard)
       "kprobe:sys_read,kprobe:my_*,kprobe:sys_write{}");
 
   auto bpftrace = get_strict_mock_bpftrace();
+  bpftrace->feature_ = std::make_unique<MockBPFfeature>(false);
+
   EXPECT_CALL(*bpftrace->mock_probe_matcher, get_symbols_from_traceable_funcs())
       .Times(1);
 
-  if (bpftrace->has_kprobe_multi())
-  {
-    ASSERT_EQ(0, bpftrace->add_probe(*probe));
-    ASSERT_EQ(3U, bpftrace->get_probes().size());
-    ASSERT_EQ(0U, bpftrace->get_special_probes().size());
+  ASSERT_EQ(0, bpftrace->add_probe(*probe));
+  ASSERT_EQ(4U, bpftrace->get_probes().size());
+  ASSERT_EQ(0U, bpftrace->get_special_probes().size());
 
-    std::string probe_orig_name =
-        "kprobe:sys_read,kprobe:my_*,kprobe:sys_write";
-    check_kprobe(bpftrace->get_probes().at(0), "sys_read", probe_orig_name);
-    check_kprobe(bpftrace->get_probes().at(1), "my_*", probe_orig_name);
-    check_kprobe(bpftrace->get_probes().at(2), "sys_write", probe_orig_name);
-  }
-  else
-  {
-    ASSERT_EQ(0, bpftrace->add_probe(*probe));
-    ASSERT_EQ(4U, bpftrace->get_probes().size());
-    ASSERT_EQ(0U, bpftrace->get_special_probes().size());
+  std::string probe_orig_name = "kprobe:sys_read,kprobe:my_*,kprobe:sys_write";
+  check_kprobe(bpftrace->get_probes().at(0), "sys_read", probe_orig_name);
+  check_kprobe(bpftrace->get_probes().at(1), "my_one", probe_orig_name);
+  check_kprobe(bpftrace->get_probes().at(2), "my_two", probe_orig_name);
+  check_kprobe(bpftrace->get_probes().at(3), "sys_write", probe_orig_name);
+}
 
-    std::string probe_orig_name =
-        "kprobe:sys_read,kprobe:my_*,kprobe:sys_write";
-    check_kprobe(bpftrace->get_probes().at(0), "sys_read", probe_orig_name);
-    check_kprobe(bpftrace->get_probes().at(1), "my_one", probe_orig_name);
-    check_kprobe(bpftrace->get_probes().at(2), "my_two", probe_orig_name);
-    check_kprobe(bpftrace->get_probes().at(3), "sys_write", probe_orig_name);
-  }
+TEST(bpftrace, add_probes_wildcard_kprobe_multi)
+{
+  ast::Probe *probe = parse_probe(
+      "kprobe:sys_read,kprobe:my_*,kprobe:sys_write{}");
+
```

Comment:
nit: extra line between var declarations

Reply:
Yeah, that's on purpose to visually separate the "mocks setting" below.